lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -603,33 +603,6 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
 
     _subscriptions.add(html.document.onSelectionChange.listen(_handleChange));
 
-    // The behavior for blur in DOM elements changes depending on the reason of
-    // blur:
-    //
-    // (1) If the blur is triggered due to tab change or browser minimize, same
-    // element receives the focus as soon as the page reopens. Hence, text
-    // editing connection does not need to be closed. In this case we dot blur
-    // the DOM element.
-    //
-    // (2) On the other hand if the blur is triggered due to interaction with
-    // another element on the page, the current text connection is obsolete so
-    // connection close request is send to Flutter.
-    //
-    // See [HybridTextEditing.sendTextConnectionClosedToFlutterIfAny].
-    //
-    // In order to detect between these two cases, after a blur event is
-    // triggered [domRenderer.windowHasFocus] method which checks the window
-    // focus is called.
-    _subscriptions.add(domElement.onBlur.listen((_) {
-      if (domRenderer.windowHasFocus) {
-        // Focus is still on the body. Continue with blur.
-        owner.sendTextConnectionClosedToFrameworkIfAny();
-      } else {
-        // Refocus.
-        domElement.focus();
-      }
-    }));
```

Comment:
As discussed in chat, we need to refocus on blur in order for the keyboard to continue to work.

Reply:
done. thanks for catching this!